roles/power_aix_bootstrap/tasks/dnf_install.yml

```diff
@@ -42,143 +42,91 @@
-          dest: "{{ download_dir }}/{{ dnf_install_script }}"
-          mode: '0755'
        delegate_to: localhost
 
      # DEFINE / EXPAND target path
-     - name: Check for target directory "{{ target_dir }}"
-       raw: "test -d {{ target_dir }} && echo true || echo false"
-       register: target_exists
-       changed_when: false
 
      - name: Create target filesystem "{{ target_dir }}" for image transfer if it does not exist.
-       raw: "crfs -v jfs2 -g rootvg -a size=350M -m {{ target_dir }} -A yes -p rw"
-       register: crfs_result
-       when: target_exists.stdout is search("false")
+       filesystem:
+          state: present
+          filesystem: "{{ target_dir }}"
+          fs_type: jfs2
+          auto_mount: yes
+          attributes: size=350M
+          permissions: 'rw'
+          vg: rootvg
 
      - name: Mount target filesystem "{{ target_dir }}"
-       raw: "mount {{ target_dir }}"
-       when: crfs_result.rc is defined and crfs_result.rc == 0
-
-
-     # TRANSFER files to target inventory
-     - name: Transfer install images if AIX 7.3
-       raw: >
-        scp -p {{ download_dir }}/{{ dnf_install_script }}
-        {{ download_dir }}/{{ dnf_src_73 }}
-        root@{{ aix_host }}:{{ target_dir }}
-       register: scp_result
-       ignore_errors: true
-       when: aix73
-       delegate_to: localhost
+       mount:
+          mount_dir: "{{ target_dir }}"
 
-#     - name: Transfer install images if AIX 7.1 or AIX 7.2
-#       raw: >
-#        scp -p {{ download_dir }}/{{ dnf_install_script }}
-#        {{ download_dir }}/{{ dnf_src_71_72 }}
-#        root@{{ aix_host }}:{{ target_dir }}
-#       register: scp_result
-#       ignore_errors: true
-#       when: not aix73
-#       delegate_to: localhost
-
-
-     # EXECUTE restore on target inventory
-     # The modified AIX toolsbox script will restore the appropiate bundle if
-     # yum is install, it will upgrade it to yum4 which is dnf with yum links and if yum is not there,
-     # it will just install dnf.
+     - name: Copy dnf install images to "{{ aix_host }}:{{ target_dir }}"
+       copy:
+          src: "{{ item }}"
+          dest: "{{ target_dir }}"
+          mode: 0755
+       with_items:
+         - "files/{{ dnf_install_script }}"
+         - "{{ download_dir }}/{{ dnf_src_73 }}"
 
      - name: Restore dnf bundle to upgrade yum to yum4 if yum is installed.
-       raw: "{{ target_dir }}/{{ dnf_install_script }} -y {{ target_dir }}"
-       ignore_errors: true
+       shell: "{{ target_dir }}/{{ dnf_install_script }} -y {{ target_dir }}"
+       failed_when: false
        when:
-          - scp_result.rc == 0
           - yum_exists.stdout is search("true")
 
      - name: Restore dnf bundle at the target if yum is not installed.
-       raw: "{{ target_dir }}/{{ dnf_install_script }} -d {{ target_dir }}"
-       ignore_errors: true
+       shell: "{{ target_dir }}/{{ dnf_install_script }} -d {{ target_dir }}"
```

Comment:
same comment as previous, use `command` instead of `shell` when not piping output